Office 365/o365/ingest/parser.yml

```diff
@@ -74,7 +74,7 @@ stages:
       - set:
           "@timestamp": "{{json_event.message.CreationTime}}"
           service.name: "{{json_event.message.Workload}}"
-          event.action: "{{json_event.message.Operation}}"
+          event.action: "{{json_event.message.Operation.rstrip('.')}}"
```

Comment:
Can you move this instruction in a `set` action protected with a filter about the value of `Operation` (If null or not)?